push/service.go

```diff
@@ -55,17 +89,34 @@ func newClient(cert tls.Certificate) (*http.Client, error) {
 	return &http.Client{Transport: transport}, nil
 }
 
-// Push notification to APN service after performing serialization.
-func (s *Service) Push(deviceToken string, headers *Headers, payload interface{}) (string, error) {
+// Push notification queues a notification APN service after performing serialization.
```

Comment:
If I expose Notification, there could be a NewNotification that does the serialization and may return an error.